Cubical/Foundations/SIP.agda

```diff
@@ -31,17 +31,28 @@ private
 -- a proposition. Indeed this type should correspond to the ways s and t can be identified
 -- as S-structures. This we call a standard notion of structure or SNS.
 -- We will use a different definition, but the two definitions are interchangeable.
+
 SNS-≡ : (S : Type ℓ₁ → Type ℓ₂) (ι : StrIso S ℓ₃) → Type (ℓ-max (ℓ-max (ℓ-suc ℓ₁) ℓ₂) ℓ₃)
 SNS-≡ {ℓ₁} S ι = ∀ {X : Type ℓ₁} (s t : S X) → ι (X , s) (X , t) (idEquiv X) ≃ (s ≡ t)
 
+SNS-≡' : (S : Type ℓ₁ → Type ℓ₂) (ι : StrHom S ℓ₃) → Type _
```

Comment:
Can you move the changes related to this to a separate PR? It would be good to have a separate discussion about these things

Reply:
By "this" I mean all changes related to StrHom